docker-compose.yml

```diff
@@ -23,6 +23,9 @@ x-rails-config: &default-rails-config
   env_file:
     # NOTE: We need this to have DATABASE_URL in bin/docker-entrypoint.
     - .env.production.local
+  environment:
+    # CAUTION: `web` overrides all env vars listed here with its own `environment` section.
+    RAILS_MAX_THREADS: 1
```

Comment:
It looks like Sidekiq wasn't respecting the `RAILS_MAX_THREADS=2` that we had in `.env.production`. So I don't think it will respect the `RAILS_MAX_THREADS=1` that we have there now. So, I'm adding this back.

![image](https://github.com/user-attachments/assets/af178617-ae47-4359-ba44-f0608ab6355d)

Reply:
We can see from the deploy that just went out after merging this that it is respected:

![image](https://github.com/user-attachments/assets/7a0d06b1-a454-4074-b2aa-735f24c1f1e9)